### PR TITLE
add guidance on the index and other structure

### PR DESCRIPTION
**Pull request by jdkandersson**
Adds guidance on the structure of the document to make it easier to contribute to the standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,13 @@ to anything in this file).
 
 The structure of the contribution guide is:
 
-* a heading to aggregate similar standards,
-* followed by one or more standards of the form:
+* Each line is limited to 80 characters for readability
+* The content index is sorted alphabetically to facilitate quickly looking up a
+  standard
+* Optionally, headings can be used to aggregate similar standards
```
Comment (arturo-seijas) [inline README.md:15]: Too many "standards" occurrences in this paragraph, could you rephrase?

Comment (jdkandersson) [reply README.md:15]: `standard` is the name of the thing being discussed, do you think there should be alternate names? Or is it useful to use the same name consistently? Or maybe `standard` isn't the right name?

Comment (arturo-seijas) [reply README.md:15]: It think it is adequate and it will be easier to read if we do not use alternate names, but I'd rephrase to something like 
```
Optionally, headings can be used to aggregate similar standards
Standards should be grouped together for readability when similar,
  which means in the body, they do not have to be sorted alphabetically
* Each standard has the structure
```

Comment (jdkandersson) [reply README.md:15]: Nice, thanks for the suggestion

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by amandahla — APPROVED** [review]
LGTM
